djsf.json

```diff
@@ -37,6 +37,11 @@
 					"description": "The type of the generated schedule (\"period\" or \"day\").",
 					"id": "#scheduletype",
 					"required":true
+				},
+				"timestamp": {

```

Comment:
I would suggest to name this `generatedAt` or `createdAt` to more specifically say what for a timestamp that is, without having to read the documentation.